tests/endtoend/eventhub_batch_functions/eventhub_multiple/function.json

```diff
@@ -0,0 +1,22 @@
+{
+  "scriptFile": "__init__.py",
+
+  "bindings": [
+    {
+      "type": "eventHubTrigger",
+      "name": "events",
+      "direction": "in",
+      "cardinality": "many",
+      "dataType": "string",
```

Comment:
do not specify dataType. String is default

Reply:
Looks like we are failing on a JSON deserialization step when `dataType` is not specified as `string`. The SDK must be assuming that we are writing a JSON deserializable string by default. I was able to repro in Node. Will open an issue against the SDK.

Reply:
Opened an issue on the EventHub extension: https://github.com/Azure/azure-functions-eventhubs-extension/issues/17.